webui/src/pages/extension-list/extension-list-header.tsx

```diff
@@ -114,40 +126,43 @@ class ExtensionListHeaderComp extends React.Component<ExtensionListHeaderComp.Pr
+                                    onChange={this.handleSearchChange}
+                                    className={classes.inputBase}
+                                    placeholder='Search for Name, Tags or Description'
+                                    id="search-input"
+                                >
+                                </InputBase>
+                                <label
+                                    htmlFor="search-input"
+                                    className="visually-hidden"
+                                >
+                                    Search for Name, Tags or Description
                             </label>
-                            <IconButton color='primary' aria-label="Search" classes={{ root: classes.iconButton }}>
-                                <SearchIcon />
-                            </IconButton>
-                        </Paper>
-                        <Paper className={classes.category}>
-                            <Select
-                                value={this.state.category}
-                                onChange={this.handleCategoryChange}
-                                renderValue={this.renderValue}
-                                displayEmpty
-                                input={<InputBase className={classes.inputBase}></InputBase>}>
-                                <MenuItem value=''>All Categories</MenuItem>
-                                {this.categories.map(c => {
-                                    return <MenuItem value={c} key={c}>{c}</MenuItem>;
-                                })}
-                            </Select>
-                        </Paper>
+                                <IconButton color='primary' aria-label="Search" classes={{ root: classes.iconButton }}>
+                                    <SearchIcon />
+                                </IconButton>
+                            </Paper>
+                            <Paper className={classes.category}>
+                                <Select
+                                    value={this.state.category}
+                                    onChange={this.handleCategoryChange}
+                                    renderValue={this.renderValue}
+                                    displayEmpty
+                                    input={<InputBase className={classes.inputBase}></InputBase>}>
+                                    <MenuItem value=''>All Categories</MenuItem>
+                                    {this.categories.map(c => {
+                                        return <MenuItem value={c} key={c}>{c}</MenuItem>;
+                                    })}
+                                </Select>
+                            </Paper>
+                        </Box>
+                        <Box className={classes.resultNumber} >( {`${this.props.resultNumber} Result${this.props.resultNumber !== 1 ? 's' : ''}`} )</Box>
```

Comment:
IMO the parentheses are a bit distracting. I would prefer `2 Results` instead of `( 2 Results )`.